docs/source/40_deploying_ADAM.md

```diff
@@ -512,4 +512,81 @@ def _count_child(job, masterHostname):
     
     # and now, count it
     assert rdd.count() == 10000
-```
\ No newline at end of file
+```
+
+## Running ADAM on Slurm
+
+For those groups with access to a HPC cluster with [Slurm](https://en.wikipedia.org/wiki/Slurm_Workload_Manager) 
+managing a number of compute nodes with local and/or network attached storage, it is possible to spin up a 
+temporary Spark cluster for use by ADAM.
+
+While the full IO bandwidth benefits of Spark processing are likely best realized through a set of 
+co-located compute/storage nodes, depending on your network setup you may find Spark deployed on HPC 
+to be a workable solution for testing or even production at scale, especially for those applications 
+which perform multiple in-memory transformations and thus benefit from Spark's in-memory processing model.
+
+Follow the primary [instructions](https://github.com/bigdatagenomics/adam/blob/master/docs/source/02_installation.md) 
+for installing ADAM into `$ADAM_HOME`
+
+### Start Spark cluster
+
+A Spark cluster can be started as a muti-node job in Slurm by creating a job file `run.cmd` such as below:
+```
```

Comment:
If you add `bash` at the end of this line, it'll do bash specific syntax supporting.